packages/react-components/react-search/src/components/SearchBox/SearchBox.test.tsx

```diff
@@ -1,19 +1,160 @@
   });
+
+  // Tests from Input, added here since they were reimplemented for SearchBox
+  it('respects value', () => {
+    renderedComponent = render(<SearchBox value="hello" />);
+    expect(getSearchBox().value).toEqual('hello');
+  });
+
+  it('respects updates to value', () => {
+    renderedComponent = render(<SearchBox value="hello" />);
+    expect(getSearchBox().value).toEqual('hello');
+
+    renderedComponent.rerender(<SearchBox value="world" />);
+    expect(getSearchBox().value).toEqual('world');
+  });
+
+  it('respects defaultValue', () => {
+    renderedComponent = render(<SearchBox defaultValue="hello" />);
+    expect(getSearchBox().value).toEqual('hello');
+  });
+
+  it('ignores updates to defaultValue', () => {
+    renderedComponent = render(<SearchBox defaultValue="hello" />);
+    expect(getSearchBox().value).toEqual('hello');
+
+    renderedComponent.rerender(<SearchBox defaultValue="world" />);
+    expect(getSearchBox().value).toEqual('hello');
+  });
+
+  it('prefers value over defaultValue', () => {
+    renderedComponent = render(<SearchBox value="hello" defaultValue="world" />);
+    expect(getSearchBox().value).toEqual('hello');
+  });
+
+  it('with value, calls onChange but does not update on text entry', () => {
+    const onChange = jest.fn();
+    renderedComponent = render(<SearchBox value="hello" onChange={onChange} />);
+    const searchBox = getSearchBox();
+    fireEvent.change(searchBox, { target: { value: 'world' } });
+    expect(onChange).toHaveBeenCalledTimes(1);
+    expect(onChange.mock.calls[0][1]).toEqual({ value: 'world' });
+    expect(searchBox.value).toBe('hello');
+  });
+
+  it('with defaultValue, calls onChange and updates value on text entry', () => {
+    const onChange = jest.fn();
+    renderedComponent = render(<SearchBox defaultValue="hello" onChange={onChange} />);
+    const searchBox = getSearchBox();
+    fireEvent.change(searchBox, { target: { value: 'world' } });
+    expect(onChange).toHaveBeenCalledTimes(1);
+    expect(onChange.mock.calls[0][1]).toEqual({ value: 'world' });
+    expect(searchBox.value).toBe('world');
+  });
+
+  it('does not call onChange when value prop updates', () => {
+    const onChange = jest.fn();
+    renderedComponent = render(<SearchBox value="hello" onChange={onChange} />);
+    renderedComponent.rerender(<SearchBox value="world" onChange={onChange} />);
+    expect(onChange).toHaveBeenCalledTimes(0);
+  });
+
+  it('hides contentAfter and dismiss at rest, shows contentAfter and dismiss when focused', () => {
+    const { container } = render(
+      <>
+        <SearchBox contentAfter={{ 'data-testid': 'contentAfter' } as React.HTMLAttributes<HTMLSpanElement>} />
+        <div data-testid="outside">outside</div>
+      </>,
+    );
+
+    const searchBox = getSearchBox();
+
+    // While unfocused
+    expect(document.activeElement).not.toEqual(searchBox);
+    expect(container.querySelector(searchBoxClassNames.dismiss)).toBeNull;
```

Comment:
`.toBeNull()` is a function, since you're not calling it here this will always return true because the function exists.

That said, I think you might actually need to test the styles, I forgot that you can't directly access the focused classname from the tests 😅